### DISCUSSION TITLE
Welcome to PasteMe Discussions!

### DISCUSSION THREAD

**Discussion by lnxpy** [collove/pasteme#44] [Announcements]
## 👋 Welcome to PasteMe!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lnxpy** [comment]
**One day, this message will remind me of the good time I was spending to develop PasteMe! :)**

**Reply by farhaduneci** [reply]
Me either ツ
